client/src/components/LandingPage.jsx

```diff
@@ -179,30 +209,55 @@ function LandingPage() {
     setFilteredSuggestions([]);
   };
 
+  const handleCalculate = () => {
+    const saveData = async () => {
+      const data = await fetch("http://localhost:5000/put-user-details", {
```

Comment:
Add `http://localhost:5000` in `.env` and use `process.env` here, later when you deploy you don't have to change domain in multiple files.